packages/react/src/primitives/shared/responsive/__tests__/useBreakpoint.test.ts

```diff
@@ -26,31 +26,10 @@ const breakpoints: Breakpoints = {
 
 const defaultBreakpoint = 'base';
 
-let matchMedia: MatchMediaMock;
-let mediaQueries = getMediaQueries({ breakpoints });
-
-const testBreakpoints = (m) => {
```

Comment:
Curious why linting didn't like sharing this function. I thought it was pretty clever ;)

Reply:
Eslint does not care abt cleverness 😭

Reply:
All jokes aside, maybe it kind of breaks jest idioms. Could see an argument for requiring the tests to be inside an `it` for readability?